mix.exs

```diff
@@ -41,8 +41,10 @@ defmodule SpandexTesla.MixProject do
   defp deps do
     [
       {:spandex, "~> 3.0", optional: true},
+      {:tesla, ">= 1.3.3", optional: true},
```

Comment:
Do we really need to add `tesla` as a dependency? Even a optional one?

Reply:
I added this to prevent people from using versions of Tesla older than 1.3.3:

<img width="577" alt="image" src="https://user-images.githubusercontent.com/677868/125301227-27f87980-e301-11eb-9539-b2b095a4707b.png">

Reply:
After the discussion on supporting earlier versions of Tesla, it makes sense to remove this dependency altogether.